api/client/search.go

```diff
@@ -27,10 +27,9 @@ func (r ByStars) Less(i, j int) bool { return r[i].StarCount < r[j].StarCount }
 // Usage: docker search [OPTIONS] TERM
 func (cli *DockerCli) CmdSearch(args ...string) error {
 	cmd := Cli.Subcmd("search", []string{"TERM"}, Cli.DockerCommands["search"].Description, true)
-	noTrunc := cmd.Bool([]string{"#notrunc", "-no-trunc"}, false, "Don't truncate output")
```

Comment:
Oh sorry, I misread the above line - I thought you were removing it - I missed that its down below on line 30. I was used to seeing git-diff showing just the deleted words.